docs/auth/oauth-popup-flow.svg

```diff
@@ -0,0 +1,56 @@
+@startuml oauth-popup-flow
+
+skinparam monochrome true
+skinparam shadowing false
+skinparam backgroundColor #fefefe

```

Comment:
covfefe

Reply:
fun fact: setting this to `white` or `#ffffff` makes the background transparent, as in `#ffffffuuuuuu`